loadgen/loadgen.cc

```diff
@@ -322,6 +322,12 @@ std::vector<QueryMetadata> GenerateQueries(
       duration_multiplier * settings.target_duration;
   size_t min_queries = settings.min_query_count;
 
+  if (scenario == TestScenario::Server) {
+    double min_queries_from_duration = DurationToSeconds(min_duration) * requested_qps;
```

Comment:
Do you mean **settings.**min_duration and **settings.**requested_qps?
And I see this compile error:
```
loadgen.cc:429:27: error: no viable overloaded '*='
        q.scheduled_delta *= scaling_value;
```